pkg/server/plugin/datastore/sql/list_attested_nodes_query_test.go

```diff
@@ -2470,7 +2469,9 @@ WHERE id IN (
 		if tt.supportsCTE {
 			name += "-cte"
 		}
-		expiresBefore := time.Now().Unix()
+		expiresBefore := time.Now()
+		bannedTrue := true
```

Comment:
Is there a better/more idiomatic way to handle boolean pointers in these tests? `&true` and `&false` are not valid, so this felt like the most straightforward alternative.

Reply:
This seems fine to me :)